repository-data/application/src/main/resources/hcm-config/hst/configurations/publicationsystem/components/legacypublication.yaml

```diff
@@ -0,0 +1,10 @@
+---
+definitions:
+  config:
+    /hst:hst/hst:configurations/publicationsystem/hst:components/legacypublication:
+      hst:componentclassname: uk.nhs.digital.ps.components.LegacyPublicationComponent
```

Comment:
Could be wrong but don't think this file is required since PAGES/legacypublication.yaml defines the componentclassname.  We do have two other duplicates i.e common/components/about.yaml, and publicationsystemcomponents/publication.yaml, so those could be removed as well at some point.